wiki/Client/Release_stream/Lazer/en.md

```diff
@@ -0,0 +1,43 @@
+---
+stub: true
+tags:
+  - client
+  - game client
+  - osu!lazer
+  - osu! lazer
+  - osu lazer
+  - osu!(lazer)
+---
+
+# Lazer release stream
+
+::: Infobox
+
+![](/img/lazer.png "The osu!(lazer) client logo, a stylised version of the osu! cookie")
```

Comment:
This logo is kinda temporary FWIW. Doesn't need to be said here, but just for your own knowledge.